lib/bamboozled/api/meta.rb

```diff
@@ -1,12 +1,22 @@
 module Bamboozled
   module API
     class Meta < Base
+      def users
+        request(:get, "meta/users").values
+      end
+
+      def fields
+        request(:get, "meta/fields")
+      end
+
+      def lists
+        request(:get, "meta/lists")
+      end
 
-      [:field, :table, :list, :user].each do |action|
-        define_method("#{action}s") do
-          result = request(:get, "meta/#{action}s")
-          result["#{action}s".to_sym][action]
-        end
+      def tables
+        request(
+          :get, "meta/tables",
+          typecast_values: false)[:tables]
```

Comment:
Layout/MultilineMethodCallBraceLayout: Closing method call brace must be on the line after the last argument when opening brace is on a separate line from the first argument.